datanode/sqlstore/migrations/0060_teams_stats.sql

```diff
@@ -0,0 +1,72 @@
+-- +goose Up
+alter table rewards
+  add column game_id bytea null;
```

Comment:
You should add an `if not exists` after the `add column` in case my game API PR gets merged before yours as I also make this update on the rewards table.

Reply:
Right, that's probably something that will disappear once yours is merged.